tests/unit/protocol.tcl

```diff
@@ -69,37 +69,37 @@ start_server {tags {"protocol network"}} {
     }
 
     set c 0
-    foreach seq [list "\x00" "*\x00" "$\x00"] {
+    foreach seq [list "\x00" "*\x00" "$\x00" "*1\r\n$\x00"] {
         incr c
         test "Protocol desync regression test #$c" {
             if {$::tls} {
                 set s [::tls::socket [srv 0 host] [srv 0 port]]
             } else {
                 set s [socket [srv 0 host] [srv 0 port]]
             }
+            fconfigure $s -blocking 0
             puts -nonewline $s $seq
+            set PROTO_INLINE_MAX_SIZE [expr 1024 * 64]
```

Comment:
Is this a hard-coded check in the implementation? It looks like it, but please add a comment about it anyway. For me (the reader) it's not easy to remember if it's a hard-coded or configurable limit.